fuzz/dex/v0.1/fuzzDexExecutorEventRemoveLiquidity.go

```diff
@@ -0,0 +1,108 @@
+package dex
+
+import (
+	"errors"
+	"fmt"
+
+	vmi "github.com/ElrondNetwork/elrond-go/core/vmcommon"
+)
+
+func (pfe *fuzzDexExecutor) removeLiquidity(user string, swapPair SwapPair, amount int, amountAmin int,
+	amountBmin int, statistics *eventsStatistics) error {
+
+	rawEquivalent, errEquivalent := pfe.querySingleResultStringAddr(pfe.ownerAddress, swapPair.address,
+		"getEquivalent", fmt.Sprintf("\"str:%s\", \"%d\"", swapPair.firstToken, 1000))
+
+	tokenABefore, err := pfe.getTokens(user, swapPair.firstToken)
+	if err != nil {
+		return nil
+	}
+	tokenBBefore, err := pfe.getTokens(user, swapPair.secondToken)
+	if err != nil {
+		return nil
+	}
+	tokenLpBefore, err := pfe.getTokens(user, swapPair.lpToken)
+	if err != nil {
+		return err
+	}
+
+	output, err := pfe.executeTxStep(fmt.Sprintf(`
+	{
+		"step": "scCall",
+		"txId": "remove-liq",
+		"tx": {
+			"from": "%s",
+			"to": "%s",
+			"value": "0",
+			"function": "removeLiquidity",
+			"esdt": {
+				"tokenIdentifier": "str:%s",
+				"value": "%d"
+			},
+			"arguments": [
+				"%d",
+				"%d"
+			],
+			"gasLimit": "100,000,000",
+			"gasPrice": "0"
+		}
+	}`,
+		user,
+		swapPair.address,
+		swapPair.lpToken,
+		amount,
+		amountAmin,
+		amountBmin,
+	))
+	if output == nil {
```

Comment:
check the error not the output, same as above (if err != nil { return err })

Reply:
done